config/overlays/kp-config.yaml

```diff
@@ -0,0 +1,43 @@
+#@ load("values.star", "values", "kp_default_docker_configjson")
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:  kp-config
+  namespace: kpack
+data:
+  default.repository: #@ values.kp_default_repository
+  default.repository.serviceaccount: kp-default-repository-serviceaccount
+  canonical.repository: #@ values.kp_default_repository
```

Comment:
When can we drop this value again?

Reply:
i think it makes sense for us to remove now